FAQs/README.md

```diff
@@ -25,6 +25,9 @@
 - ### Control Remediation
  1. [I want to turn OFF public access on key vault, but it is consumed by Azure Function Apps using consumption plan, How do I add the IP Addresses which are dynamically changing?](#1-i-want-to-turn-off-public-access-on-key-vault-but-it-is-consumed-by-azure-function-apps-using-consumption-plan-how-do-i-add-the-ip-addresses-which-are-dynamically-changing)
```

Comment:
How can I disable public access on my Key Vault when it is being used by Azure Function Apps on a consumption plan? How do I handle dynamically changing IP addresses?

Reply:
Not updating this as of now as it might break links where we have shared this FAQ previously. Please let me know if I should still update

Reply:
Seems this particular FAQ link was not shared for many instances, Updated as suggested